OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -52,12 +52,21 @@ public class HarvesterInfo : ITraitInfo, Requires<MobileInfo>
 		[Desc("Automatically scan for resources when created.")]
 		public readonly bool SearchOnCreation = true;
 
-		[Desc("Initial search radius (in cells) from the refinery that created us.")]
-		public readonly int SearchFromProcRadius = 24;
+		[Desc("Initial search radius (in cells) from the refinery (or factory) that created us.",
+			"Also used to perform fallback scan if nothing is found within SearchFromOrderRadius.")]
+		public readonly int SearchFromRefineryRadius = 24;
 
 		[Desc("Search radius (in cells) from the last harvest order location to find more resources.")]
 		public readonly int SearchFromOrderRadius = 12;
 
+		[Desc("AI override for initial search radius (in cells) from the refinery (or factory) that created us.",
+			"Negative values make it default to the normal SearchFromRefineryRadius.")]
+		public readonly int SearchFromRefineryRadiusAIOverride = -1;
```

Comment:
Can we drop the "Override" from these names for consistency with `InitialStanceAI` in `AutoTarget`?